docs/shader-modules/project.md

```diff
@@ -95,15 +95,27 @@ Projects world space coordinates to clipspace, which can be assigned to `gl_Posi
 `vec4 project_to_clipspace(vec4 position)`
 
 
-### project_pixels_to_clipspace
+### project_pixel_to_clipspace
```

Comment:
Nit: While I get the temptation to clean up the entire API when making changes, have to wonder if this one-character change is valuable enough that it is worth the extra fiddling it will cause for writers of custom layers.

Reply:
This function was not renamed. It was a typo in the docs.